src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -181,3 +193,20 @@ interface TriggeredInfo {
     val user: User?
     val build: Build?
 }
+
+interface Trigger{
```

Comment:
How this interface is supposed to be used? What values 'type' property can have? Which parameters different triggers have? With the current implementation it's impossible to use Trigger interface without digging into TeamCity rest documentation.

This library is supposed to provide convenient access to TeamCity settings, not just dummy Kotlin wrapper around json structures. It's better to add interfaces for concrete build triggers with corresponding properties instead of adding a generic Trigger interface. You can start with a type of trigger you're interested in, and add other types later.